Caboodle/PhoneDialer/PhoneDialer.ios.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using Foundation;
+using UIKit;
+using CoreTelephony;
+
+namespace Microsoft.Caboodle
+{
+	public static partial class PhoneDialer
+	{
+		public static bool IsSupported
+		{
+			get
+			{
+				var nsUrl = CreateNsUrl("0000000000");
+				var canCall = UIApplication.SharedApplication.CanOpenUrl(nsUrl);
+				
+				if (canCall)
+				{
+					using (var netInfo = new CTTelephonyNetworkInfo())
+					{
+						var mnc = netInfo.SubscriberCellularProvider?.MobileNetworkCode;
+						return !string.IsNullOrEmpty(mnc) && mnc != "65535"; // 65535 stands for NoNetwordProvider
```

Comment:
don't need to check against null here.

Reply:
All these properties might be null.
https://developer.apple.com/documentation/coretelephony/cttelephonynetworkinfo/1616900-subscribercellularprovider
https://developer.apple.com/documentation/coretelephony/ctcarrier/1620324-mobilenetworkcode